stan/math/gpu/copy.hpp

```diff
@@ -107,13 +159,16 @@ inline void copy(matrix_gpu& dst, const matrix_gpu& src) {
                    dst.cols());
   if (src.size() > 0) {
     try {
+      cl::CommandQueue queue = opencl_context.queue();
       /**
        * Copies the contents of the src buffer to the dst buffer
        * see the matrix_gpu(matrix_gpu&) constructor
        *  for explanation
        */
-      opencl_kernels::copy(cl::NDRange(dst.rows(), dst.cols()), src.buffer(),
-                           dst.buffer(), dst.rows(), dst.cols());
+      cl::Event copy_event;
```

Comment:
What motivated the switch from the copy kernel? Just curious, no opinion here.

Reply:
meant to ping @rok-cesnovar about this. Was our copy faster than OpenCL's version? If not then I'd like to change to the normal version

Reply:
Yep, this way is actually faster. Doing a PR that removes the copy kernel code as well as replaces the zeros kernel with enqueueFillBuffer is on my short list of small updates we have to do after we get the Cholesky stuff through.